src/strings-tasks.js

```diff
@@ -286,10 +339,19 @@ function formatTime(/* minutes, seconds */) {
  *   reverseString('abcdef') => 'fedcba'
  *   reverseString('12345') => '54321'
  */
-function reverseString(/* str */) {
-  throw new Error('Not implemented');
+// function reverseString(/* str */) {
+//   throw new Error('Not implemented');
+// }
+
+function reverseString(str) {
+  const splitString = str.split('');
+  const reverseArray = splitString.reverse();
+  const joinArray = reverseArray.join('');
+  return joinArray;
```

Comment:
```suggestion
  return str.split('').reverse().join('');
```
Тут можно через точку объединить все функции. И вообще в js есть такая возможность, не обязательно каждый раз объявлять переменную